### PR TITLE
Kill pseudo toplevel methods

### PR DESCRIPTION
**Pull request by privat**
Move some methods from Object to the top-level (Sys) as they do not rely on `self` and are intended to be called from everywhere.

Beside documentation improvement, this should slightly reduce the size of tables as methods introduced in Object are inherited by all classes.

### CODE REVIEW

File: contrib/wiringPi/lib/wiringPi.nit [hunk 1]
```diff
@@ -29,20 +29,21 @@ in "C Header" `{
...
-	# use the Broadcom GPIO pin numbers directly with no re-mapping.
-	fun wiringPi_setup_gpio `{ wiringPiSetupGpio(); `}
-	# Identical to wiringPi_setup, however it allows the calling
-	# programs to use the physical pin numbers on the P1 connector only.
-	fun wiringPi_setup_phys `{ wiringPiSetupPhys(); `}
-	# This initialises wiringPi but uses the /sys/class/gpio interface
-	# rather than accessing the hardware directly.
-	fun wiringPi_setup_sys `{ wiringPiSetupSys(); `}
-end
+# Initialises wiringPi.
```
Comment (Morriar) [inline contrib/wiringPi/lib/wiringPi.nit]: Initializes*

File: contrib/wiringPi/lib/wiringPi.nit [hunk 2]
```diff
@@ -29,20 +29,21 @@ in "C Header" `{
...
+
+# Same as wiringPi_setup, but with GPIO.
+# It allows the calling programs to use the Broadcom GPIO pin numbers directly with no re-mapping.
+fun wiringPi_setup_gpio `{ wiringPiSetupGpio(); `}
+
+# Same as to wiringPi_setup, but with physical pin numbers.
+# It allows the calling programs to use the physical pin numbers on the P1 connector only.
+fun wiringPi_setup_phys `{ wiringPiSetupPhys(); `}
+
+# This initialises wiringPi, but with the /sys/class/gpio interface.
```
Comment (Morriar) [inline contrib/wiringPi/lib/wiringPi.nit]: Initializes

**Comment by Morriar** [comment]
+1

**Comment by privat** [comment]
reroll: apply spell and try to pass tests

**Comment by xymus** [comment]
+1

**Pull request merged**